service/src/main/java/uk/gov/hmcts/reform/fpl/enums/docmosis/RenderFormat.java

```diff
@@ -2,13 +2,28 @@
 
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
+import org.apache.commons.io.FilenameUtils;
+
+import java.util.Arrays;
 
 @Getter
 @RequiredArgsConstructor
 public enum RenderFormat {
     WORD("doc", "application/msword"),
+    DOC_X("docx", "application/vnd.openxmlformats-officedocument.wordprocessingml.document"),
     PDF("pdf", "application/pdf");
 
     private final String extension;
     private final String mediaType;
+
+    public static RenderFormat fromFileName(String filename) {
+        String extension = FilenameUtils.getExtension(filename);
+        return Arrays.stream(RenderFormat.values())
+            .filter(format -> extension.equals(format.getExtension()))
+            .findFirst()
+            .orElseThrow(
```

Comment:
would it be worth returning `application/octet-stream` by default, could maybe log a warning as well if you do decide to

Reply:
The flow only allows Doc, docx and pdfs, so for this one we want to return an exception, pdf converter won't accept anything else